services/coincap/coincap-priceusd.service.js

```diff
@@ -0,0 +1,45 @@
+import Joi from 'joi'
+import BaseCoincapService from './coincap-base.js'
+
+const schema = Joi.object({
+  data: Joi.object({
+    priceUsd: Joi.string()
+      .pattern(/[0-9]*\.[0-9]+/i)
+      .required(),
+    name: Joi.string().required(),
+  }).required(),
+}).required()
+
+export default class CoincapPriceUsd extends BaseCoincapService {
+  static route = { base: 'coincap/priceUsd', pattern: ':assetId' }
```

Comment:
Sorry. This is another one I didn't spot the first time. In general the hard-coded parts of the route are snake-case all lowercase. So `price-usd` here and `change-percent-24-hr` for the other one.
The params can stay as they are because they get turned into variables.